packages/ui/FloatingLayer/FloatingLayerDecorator.js

```diff
@@ -1,5 +1,10 @@
 /*
- * Exports the {@link ui/FloatingLayer.FloatingLayerDecorator} higher-order component (HOC).
+ * A Higher-order Component that adds a FloatingLayer adjacent to wrapped component.
```

Comment:
Higher-order component should be lowercase, not uppercase.

Reply:
Should we update this too as well?
https://github.com/enactjs/enact/blame/master/docs/contributing/documentation.md#L85

Reply:
Yes, please!